sympy/utilities/source.py

```diff
@@ -5,8 +5,9 @@
 from __future__ import print_function, division
 
 import inspect
+from sympy.core.decorators import deprecated
 
-
+@deprecated(issue=14908, deprecated_since_version="1.1")
```

Comment:
The most recent release is 1.2. This should have 1.3.

Reply:
Also, add the useinstead flag.

Reply:
should I change it now?

Reply:
and what to fill in useinstead flag argument?